apps/vr-tests-react-components/src/stories/RatingDisplay.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { CircleRegular, CircleFilled, SquareRegular, SquareFilled } from '@fluentui/react-icons';
+import { RatingDisplay } from '@fluentui/react-rating-preview';
+import { TestWrapperDecoratorFixedWidth } from '../utilities/TestWrapperDecorator';
+
+storiesOf('RatingDisplay Converged Size', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addStory('RatingDisplay size small', () => <RatingDisplay size="small" value={3.5} />, {})
+  .addStory(
+    'RatingDisplay size small with value and count',
```

Comment:
No need to include `RatingDisplay` on any of these story names (it's already in the `storiesOf` name)
```suggestion
    'size small with value and count',
```